crates/loans/src/lib.rs

```diff
@@ -1128,114 +1125,41 @@ impl<T: Config> Pallet<T> {
         T::PalletId::get().into_account_truncating()
     }
 
-    fn get_lf_borrowed_value(_account: &T::AccountId) -> Result<FixedU128, DispatchError> {
-        Ok(FixedU128::zero())
-    }
-
-    fn get_lf_base_position(_account: &T::AccountId) -> Result<FixedU128, DispatchError> {
-        Ok(FixedU128::zero())
-    }
-
-    fn get_lf_liquidation_base_position(_account: &T::AccountId) -> Result<FixedU128, DispatchError> {
-        Ok(FixedU128::zero())
-    }
-
-    pub fn get_account_liquidity(
-        account: &T::AccountId,
-    ) -> Result<(Liquidity, Shortfall, Liquidity, Shortfall), DispatchError> {
+    pub fn get_account_liquidity(account: &T::AccountId) -> Result<(Liquidity, Shortfall), DispatchError> {
         let total_borrow_value = Self::total_borrowed_value(account)?;
         let total_collateral_value = Self::total_collateral_value(account)?;
-        let lf_borrowed_value = Self::get_lf_borrowed_value(account)?;
-        let lf_base_position = Self::get_lf_base_position(account)?;
 
         log::trace!(
             target: "loans::get_account_liquidity",
-            "account: {:?}, total_borrow_value: {:?}, total_collateral_value: {:?}, lf_borrowed_value: {:?}, lf_base_position: {:?}",
+            "account: {:?}, total_borrow_value: {:?}, total_collateral_value: {:?}",
             account,
             total_borrow_value.into_inner(),
             total_collateral_value.into_inner(),
-            lf_borrowed_value.into_inner(),
-            lf_base_position.into_inner(),
         );
-        match (
-            total_collateral_value > total_borrow_value,
-            lf_base_position > lf_borrowed_value,
-        ) {
-            // TODO: simplify this function and use checked math operations
-            (true, true) => Ok((
-                total_collateral_value - total_borrow_value,
-                FixedU128::zero(),
-                lf_base_position - lf_borrowed_value,
-                FixedU128::zero(),
-            )),
-            (true, false) => Ok((
-                total_collateral_value - total_borrow_value,
-                FixedU128::zero(),
-                FixedU128::zero(),
-                lf_borrowed_value - lf_base_position,
-            )),
-            (false, true) => Ok((
-                FixedU128::zero(),
-                total_borrow_value - total_collateral_value,
-                lf_base_position - lf_borrowed_value,
-                FixedU128::zero(),
-            )),
-            (false, false) => Ok((
-                FixedU128::zero(),
-                total_borrow_value - total_collateral_value,
-                FixedU128::zero(),
-                lf_borrowed_value - lf_base_position,
-            )),
+        if total_collateral_value > total_borrow_value {
+            Ok((total_collateral_value - total_borrow_value, FixedU128::zero()))
+        } else {
+            Ok((FixedU128::zero(), total_borrow_value - total_collateral_value))
         }
```

Comment:
Probably would use checked or saturated math here

Reply:
Done and I'll create an issue, there are other places without failsafe math too